R/binary_classification.R

```diff
@@ -76,3 +76,44 @@ ll <- function(actual, predicted) {
 logLoss <- function(actual, predicted) {
     return(mean(ll(actual, predicted)))
 }
+
+
+
+#' Precision
+#' 
+#' \code{precision} computes proportion of predicted 1's that are actual 1's
```

Comment:
Can you change this to, "computes proportion of observations predicted to be in the positive class (i.e. the element in \code{predicted} equals 1) that actually belong to the positive class (i.e. the element in \code{actual} equals 1)?

Thanks